extensions-contrib/statsd-emitter/src/main/java/org/apache/druid/emitter/statsd/StatsDEmitter.java

```diff
@@ -174,6 +177,59 @@ public void emit(Event event)
       } else {
         log.debug("Service=[%s], Metric=[%s] has no StatsD type mapping", service, metric);
       }
+    } else if (event instanceof AlertEvent && config.isDogstatsd()) {
+      AlertEvent alertEvent = (AlertEvent) event;
+
+      ImmutableMap.Builder<String, String> tagBuilder = ImmutableMap.builder();
+
+      tagBuilder
+          .put("feed", alertEvent.getFeed())
+          .put("service", alertEvent.getService())
+          .put("severity", alertEvent.getSeverity().toString());
+      if (config.getIncludeHost()) {
+        tagBuilder.put("hostname", alertEvent.getHost());
```

Comment:
If you're changing a `"hostname"` to TAG_HOSTNAME above, do it here too?

Reply:
thanks, missed that one